apps/server/src/modules/legacy-school/controller/legacy-school.controller.ts

```diff
@@ -10,14 +10,17 @@ import { ICurrentUser } from '@src/modules/authentication';
 import { Authenticate, CurrentUser } from '@src/modules/authentication/decorator/auth.decorator';
 import { MigrationMapper } from '../mapper/migration.mapper';
 import { OauthMigrationDto } from '../uc/dto/oauth-migration.dto';
-import { SchoolUc } from '../uc/school.uc';
+import { LegacySchoolUc } from '../uc';
 import { MigrationBody, MigrationResponse, SchoolParams } from './dto';
 
+/**
+ * @deprecated because it uses the deprecated LegacySchoolDo.
+ */
 @ApiTags('School')
 @Authenticate('jwt')
 @Controller('school')
-export class SchoolController {
-	constructor(private readonly schoolUc: SchoolUc, private readonly migrationMapper: MigrationMapper) {}
+export class LegacySchoolController {
```

Comment:
Please run server client generator in nuxt repo to check generated naming

Reply:
Did so and linked the PR.